rancher/installing-rancher/installing-server/multi-nodes/index.md

```diff
@@ -11,12 +11,16 @@ _Available as of v1.0.1_
 ### Requirements 
 
 * Nodes to be used in HA setup that meet the single node [requirements]({{site.baseurl}}/rancher/installing-rancher/installing-server/#requirements) 
+    * Ports that need to be opened on Nodes
+        * Global Access: `22`, `80`, `443`, `18080` (Port exposed to [view the management stack]({{site.baseurl}}/rancher/faqs/server/#ha-monitoring) as it comes up)
```

Comment:
specify 22/tcp etc would be helpful
also, 18080/tcp is optional and only necessary to debug system stack